vmdb/db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Provisioning/StateMachines/Methods.class/__methods__/provision.rb

```diff
@@ -7,8 +7,10 @@
 service = $evm.root["service_template_provision_task"].destination
 
 begin
-  service.deploy_orchestration_stack
+  ems_ref = service.deploy_orchestration_stack
+  $evm.log("info", "Stack #{service.stack_name} with reference id (#{ems_ref}) is being created")
```

Comment:
:+1: Logging the ref returned from deploy_orchestration_stack.  Not sure it applies here, but printing a empty string and nil would look the same and is sometimes an important distinction.  I like to use `.inspect` on these types of variables so it is clear in the log if the value is a string, number or nil for clarity.